modules/installation/pages/uyuni-server-setup.adoc

```diff
@@ -0,0 +1,174 @@
+
+. Fill in the Account Information fields including an email for system notifications.
+
+. Click btn:[Create Organization] to finish creating your administration account.
++
+
+// image::quickstart-mgr-setup-admin1.png[scaledwidth=80%]
+
+You are now presented with the {productname} menu:Home[Overview] page.
+// In the next section you will prepare the server for connecting the first client.
+
+
+== Optional: Synchronizing Products from {scc}
+
+{scc} (SCC) maintains a collection of repositories which contain packages, software and updates for all supported enterprise client systems.
+These repositories are organized into channels each of which provide software specific to a distribution, release, and architecture.
+After synchronizing with SCC clients may receive updates, and be organized into groups and assigned to specific product software channels.
+
+This section covers synchronizing with SCC from the {webui} and adding your first client channel.
+
+[NOTE]
+====
+For Uyuni, synchronizing products from {scc} is optional.
+====
+
+Before you can synchronize software repositories with SCC, you will need to enter organization credentials in {productname}.
+In previous versions, so-called mirror credentials were used instead.
+The organization credentials give you access to the SUSE product downloads.
+You will find your organization credentials in https://scc.suse.com/organization.
+
+Enter your organization credentials in the {productname} {webui}:
+
+
+[[proc-admin-organization-credentials]]
+.Procedure: Entering Organization Credentials
+. In the {susemgr} {webui}, select menu:Main Menu[Admin > Setup Wizard].
+. From the [guimenu]``Setup Wizard`` page select the btn:[Organization Credentials] tab.
+. Click btn:[Add a new credential].
+. In the dialog, enter [guimenu]``Username`` and [guimenu]``Password``, and confirm with btn:[Save].
+
+When the credentials are confirmed with a check-mark icon, proceed with <<proc-quickstart-first-channel-sync>>.
+
+
+[[proc-quickstart-first-channel-sync]]
+.Procedure: Synchronizing with {scc}
+. In the {webui}, navigate to menu:Admin[Setup Wizard].
+
+. From the [guimenu]``Setup Wizard`` page select the btn:[SUSE Products] tab.
+Wait a moment for the products list to populate.
+If you previously registered with {scc} a list of products will populate the table.
+This table lists architecture, channels, and status information.
+For more information, see xref:reference:admin/setup-wizard.adoc[Wizard].
++
+
+image::admin_suse_products.png[scaledwidth=80%]
+
+. If your {sle} client is based on [systemitem]``x86_64`` architecture scroll down the page and select the check box for this channel now.
++
+
+* Add channels to {productname} by selecting the check box to the left of each channel.
+Click the arrow symbol to the left of the description to unfold a product and list available modules.
+* Click btn:[Add Products] to start product synchronization.
```

Comment:
Why did we shift to bullets here?

Reply:
The "Click the arrow symbol" sentence belongs to the "Add channels" item. If the user does not see the product, the user might appreciate if we say that using the arrow symbol could be useful.

Is this what you want to know?